### PR TITLE
Adding role to install shinken

### PR DESCRIPTION
**Pull request by kedark3**
Adding new Ansible Role for Shinken installation.

### CODE REVIEW

File: deplo_shinken_on_vmware_vm.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+- name: Create VM on VMware
+  hosts: localhost
+  connection: local
+  roles:
+    - vmware_deploy_vms
+
+- name: Deploy Shinken on vms created by previous play
+  hosts: vmware_vms
+  roles:
+    - deploy_shinken_server
```
Comment (john-dupuy) [inline deplo_shinken_on_vmware_vm.yml]: New line needed here and on other files

**Review by john-dupuy — COMMENTED** [review]
Awesome work KK, this is very cool! :tada: :tada: :+1: , ping me when the README is updated and I can take a look.

File: roles/deploy_shinken_server/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+Deploy Shinken Server
+=========
+
+This is an Ansible Role to Deploy Shinken Monitoring Server with Basic installation.
+
+Requirements
+------------
+
+CentOS 7 server is tested for this Ansible Role. It may work with Fedora or RHEL 7 given that correct repos are added on it.
```
Comment (john-dupuy) [inline roles/deploy_shinken_server/README.md:12]: Perhaps `tested and recommended` ?

File: roles/deploy_shinken_server/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```
+shinken_packs_from_git_repo:
+  - repo_name: repo1
+    repo_url: https://github.com/user/repo1
+  - repo_name: repo2
+    repo_url: https://github.com/user/repo2
+  and so on.
+```
+
+`enforce_ipv4_check_host_alive:`  This variable dictates if you want to enforce IPv4 ping or ping6 for IPv6, as when Shinken installs, it adds itself as a host and tries to perform `check_host_alive` check which will then be used to perform ping checks against itself. If you don't have IPv6 configured correctly, please user IPv4 by setting following variable to true.
```
Comment (john-dupuy) [inline roles/deploy_shinken_server/README.md]: small typo `please use IPv4 by setting this variable to true.`

File: roles/deploy_shinken_server/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+Deploy Shinken Server
+=========
+
+This is an Ansible Role to Deploy Shinken Monitoring Server with Basic installation.
```
Comment (john-dupuy) [inline roles/deploy_shinken_server/README.md]: Perhaps describe a bit what the basic installation constitutes (i.e. installs packs for vmware, rhv, linux, etc.), and maybe provide a link to the Shinken documentation so the user knows where to go after the installation completes (and how to make use of the packs).

**Review by john-dupuy — APPROVED** [review]
Some small suggestions, but awesome stuff!!

File: roles/deploy_shinken_server/README.md (hunk 2 above)
Comment (kedark3) [reply roles/deploy_shinken_server/README.md:12]: @john-dupuy, I think its not really recommended, it should work on all 3 RHEL type distros in same way. Hence gonna keep just `tested`.

**Pull request merged**